packages/react-noop-renderer/src/createReactNoop.js

```diff
@@ -227,6 +221,9 @@ function createReactNoop(reconciler: Function, useMutation: boolean) {
     },
 
     shouldSetTextContent(type: string, props: Props): boolean {
+      if (type === 'errorInBeginPhase') {
+        throw new Error('Error in host config.');
+      }
```

Comment:
Putting it here because it fires for every host node (and gives me the type). No practical difference for existing tests that just tried to mount a span.

Reply:
Why the type rather than a prop?

Reply:
To make it jump out more